docs/drafts/developer.md

```diff
@@ -69,20 +63,11 @@ To undeploy the controller from the cluster:
 make undeploy
 ```
 
-### How it works
-This project aims to follow the Kubernetes [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/).
-
-It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/)
-which provide a reconcile function responsible for synchronizing resources until the desired state is reached on the cluster.
+---
 
-### Test It Out
+## Iterative Development
 
-Install the CRDs and the operator-controller into a new [KIND cluster](https://kind.sigs.k8s.io/):
-```sh
-make run
-```
-This will build a local container image of the operator-controller, create a new KIND cluster and then deploy onto that cluster.
-This will also deploy the catalogd and cert-manager dependencies.
+If you are making code changes and want to iterate on them quickly, there are specific targets for this purpose.
 
 ### Modifying the API definitions
```

Comment:
Can we move this section below Tilt section? I do not think anyone who want to do iterative development want to see see `make manifests` first

Reply:
I am moving the section up into the manual step-by-step section along with its explanation about using it for API code changes. And eliminating the section it was in by itself before the Tilt section. Make sense?

Reply:
done in 7f54af6c

Reply:
seem ok in 7f54af6c ?